Lib/gftools/instancer.py

```diff
@@ -0,0 +1,42 @@
+from fontTools.varLib.instancer import instantiateVariableFont, OverlapMode
+from gftools.fix import update_nametable, fix_fs_selection, fix_mac_style
+
+
+__all__ = ["gen_static_font"]
+
+
+def gen_static_font(
+    var_font, family_name, style_name, axes, keep_overlaps=False, dst=None
+):
```

Comment:
The signature could be `gen_static_font(var_font, axes, family_name=None, style_name=None, keep_overlaps=Fase, dst=None)` considering instantiateVariableFont() will have a `family_name` parameter in https://github.com/fonttools/fonttools/pull/2189.

Should the `axes` argument be called `axis_values`?

Reply:
The fonttools PR will update the name table but there's a chance it won't comply with our specification. At GF, if a static family features axes which aren't wght or ital, it will be released as a [new family](https://fonts.google.com/?query=condensed). The fonttools pr doesn't do this. It will simply update nameIDs 2 and 17 and leave the family names intact.

I've just checked our [update_nametable](https://github.com/googlefonts/gftools/blob/main/Lib/gftools/fix.py#L375) and it's capable of updating the family name if non wght/ital particles exist. This means I can update the signature as you're suggesting.

Reply:
Thanks.

I've updated the signature as per your recommendation. However, we still need to update the name table using our own name table updater since GF style names can only have wght+ital tokens. The fontTools instancer doesn't have this restriction.

> Should the axes argument be called axis_values?

This may cause some confusion since some users may think they need to supply STAT AxisValues, which isn't what we're doing. Perhaps `axis_coordinates` is a better arg name?

Tests are failing because we've just merged the fontTools PR. Once a new release is cut, the tests should pass.